README.md

```diff
@@ -211,10 +211,10 @@ python -m semantic_model_generator.validate_model \
 
 ## Python
 
-You may also create/validate your semantic models from directly within your Python code. First, ensure that you have installed the Python package. Note, the version below should be the latest version under the `dist/` directory.
+You may also create/validate your semantic models from directly within your Python code. First, ensure that you have installed the Python package.
 
 ```bash
-pip install dist/*.whl
+pip install .
```

Comment:
Could we also modify the instructions in the `## Release` section at the bottom of the README? We could remove steps 4 and 6 since the whl building is no longer necessary.

Reply:
This is a great call! We may also be able to remove those steps because that seems to be taken care of in the [release](https://github.com/Snowflake-Labs/semantic-model-generator/blob/1e62bc7adda5aa058a3990c1a0124a8cca8b2e4b/.github/workflows/release.yaml) workflow.